sdk/python/kfp/_client.py

```diff
@@ -156,10 +158,9 @@ def __init__(self,
             KF_PIPELINES_APP_OAUTH2_CLIENT_ID_ENV)
         other_client_secret = other_client_secret or os.environ.get(
             KF_PIPELINES_APP_OAUTH2_CLIENT_SECRET_ENV)
-
         config = self._load_config(host, client_id, namespace, other_client_id,
                                    other_client_secret, existing_token, proxy,
-                                   ssl_ca_cert, kube_context, credentials)
+                                   ssl_ca_cert, kube_context, credentials,verify_ssl)
```

Comment:
nit: space

Please format the code following the contributing guide: https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#sdk